src/xinterpreter.cpp

```diff
@@ -138,6 +138,9 @@ __get_cxx_version ()
     {
         nl::json kernel_res;
 
+
+        auto input_guard = input_redirection(allow_stdin);
```

Comment:
Why adding this line in this PR? This seems unrelated.

Reply:
This is to fix a warning which comes when building xeus-cpp. Without it allow_stdin is an unused variable. I coped how its used in xeus-python.